src/app/tests/suites/TestIcdManagementCluster.yaml

```diff
@@ -221,6 +225,23 @@ tests:
                 value: 1001
               - name: "Key"
                 value: "\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f"
+              - name: "ClientType"
+                value: ClientTypeEnum.UnknownEnumValue
```

Comment:
You could also consider sending some other out-of-range value if the YAML will let you (like 3 or something).  That should also result in ConstraintError.

Reply:
done

Reply:
@bzbarsky-apple When adding the test case with a value > UnknownEnumValue, yaml parsing fails since it checks the valid values of the enum. Couldn't add the test case with the yaml test.

This will need to go in the python test - will do in a follow up.